packages/vue-components/src/TabGroup.vue

```diff
@@ -84,5 +88,9 @@ export default {
             margin-bottom: 10px;
             text-decoration: underline;
         }
+
+        .printable-tab-group-header.tab-group-print-block {
```

Comment:
> @ang-zeyu Thanks for the suggestion 👍
> 
> Have changed `v-show` and `d-print-block/flex` to our own custom classes. Needed to add them individually to `tab-group` and `tab` so that we can increase the specificity from there. `no-print` is added to the global css file with `!important` so that it can be reused by other components as well once we have changed out all the other usages of `!important`.

hmm... not sure I follow, what's the addition `tab-group-print-block` for? (the variants in `tab.vue` as well)

Reply:
Ahh seems like css that appear the latest will take effect anyway if same level of specificity. Initially added the `tab-group-print-block` so that it can have higher specificity over the `dispaly: none` rules to ensure that it is always overriding it.

Reply:
> Ahh seems like css that appear the latest will take effect anyway if same level of specificity.

yes, one of the main reasons its named *cascading* style sheets. Let's remove it then!